VimeoUpload/Upload/Descriptor System/Cameo/CAMUploadDescriptor.swift

```diff
@@ -10,21 +10,21 @@ import Foundation
 import AFNetworking
 import VimeoNetworking
 
-public class CAMUploadDescriptor: ProgressDescriptor, VideoDescriptor
+@objc public class CAMUploadDescriptor: ProgressDescriptor, VideoDescriptor
 {
     let videoUrl: URL
     let videoSettings: VideoSettings?
     let thumbnailUrl: URL?
     
-    public private(set) var uploadTicket: VIMUploadTicket?
-    public private(set) var pictureTicket: VIMThumbnailUploadTicket?
-    public private(set) var picture: VIMPicture?
+    @objc public private(set) var uploadTicket: VIMUploadTicket?
+    @objc public private(set) var pictureTicket: VIMThumbnailUploadTicket?
+    @objc public private(set) var picture: VIMPicture?
     
     private var currentRequest = CAMUploadRequest.CreateVideo
     {
         didSet
         {
-            print("\(self.currentRequest.rawValue) \(self.identifier)")
+            print("\(self.currentRequest.rawValue) \(String(describing: self.identifier))")
```

Comment:
I don't know whether we need to print these values at all, but if so maybe we can unwrap that identifier? Or just insert an empty string if it's nil?

Reply:
I think that print statement should be removed but for purpose of this PR I'll keep it there and use a default if we can't unwrap `identifier`.